AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContext.h

```diff
@@ -0,0 +1,28 @@
+#import <Foundation/Foundation.h>
```

Comment:
This import is not necessary.

Reply:
This contains an `NSString` property, so I believe we need the import. Unless we forward declare the `NSString` class. But that's pretty unusual.